CHANGELOG.md

```diff
@@ -1,3 +1,74 @@
+
+### Android(Espresso)
+
+* Add to raise `InvalidContextError` error when the app under test stops because of instrumentation process crashes (appium-espresso-driver#591)[https://github.com/appium/appium-espresso-driver/pull/591]
+* Add a possibility to allow JSON formatted file as `espressoBuildConfig` capability [appium-espresso-driver#609](https://github.com/appium/appium-espresso-driver/pull/609)
+* Add `mobile:` functions:
+  * `mobile:registerIdlingResources`, `mobile:unregisterIdlingResources` and `mobile:listIdlingResources` to customize idling resources [appium-espresso-driver#597](https://github.com/appium/appium-espresso-driver/pull/597)
+* Change to raise `InvalidElementStateException` error instead of `AppNotIdleException` if the app under test is not idling [appium-espresso-driver#589](https://github.com/appium/appium-espresso-driver/pull/589)
+* The `com.google.android.material` dependency became optional [appium-espresso-driver#616](https://github.com/appium/appium-espresso-driver/pull/616)
+  * Now, `espressoBuildConfig` has two parts, `additionalAppDependencies` and `additionalAndroidTestDependencies`. Please check [caps](https://appium.io/docs/en/writing-running-appium/caps/).
+  * `mobile:navigateTo` requires `com.google.android.material`, so please add `"espressoBuildConfig": "{\"additionalAndroidTestDependencies\":[\"com.google.android.material:material:1.2.1\"]}"` as your capabilities if needed
+* Fix to exclude transitive dependencies of espresso-contrib[appium-espresso-driver#596](https://github.com/appium/appium-espresso-driver/pull/596)
+* Fix to be able to build with SDK 29 [appium-espresso-driver#604](https://github.com/appium/appium-espresso-driver/pull/604)
+
+### iOS(general)
+
+* Improves state check of simulators [appium-ios-simulator#284](https://github.com/appium/appium-ios-simulator/pull/284)
+* Fix privacy for calendar access on simulators [appium-ios-simulator#288](https://github.com/appium/appium-ios-simulator/pull/288)
+* Adjust the decimal separator in the Simulator location setting script according to system locale settings [appium-ios-simulator#295](https://github.com/appium/appium-ios-simulator/pull/295)
+
+### iOS(XCUITest)
+
+* Add capabilities:
+  * `simulatorDevicesSetPath` allows to set an alternative path to the simulator devices [appium-ios-simulator#290](https://github.com/appium/appium-ios-simulator/pull/290)
+  * `allowProvisioningDeviceRegistration` adds `-allowProvisioningUpdates` and `-allowProvisioningDeviceRegistration` flag in xcodebuild [appium-xcuitest-driver#1241](https://github.com/appium/appium-xcuitest-driver/pull/1241)
+* Add mobile functions:
+  * `mobile:resetPermission` to reset all authorizations [https://github.com/appium/appium-xcuitest-driver#1239](https://github.com/appium/appium-xcuitest-driver/pull/1239), [WebDriverAgent#392](https://github.com/appium/WebDriverAgent/pull/392)
```

Comment:
to reset all previously allowed or denied permissions for the application under test (only since Xcode 11.4)